.gitignore

```diff
@@ -0,0 +1,10 @@
+.DS_Store
+*.class
+*.classpath
+*.settings
+*.project
+target/
+*.iml
+.idea/
+test_*.properties
+.sonar
```

Comment:
this file can be as simple as one line with `target/`. I'd remove at least `*.class`, `test_*.properties` and `.sonar` from this list..